scripts/integration/Dockerfile

```diff
@@ -2,5 +2,5 @@ ARG RUST_VERSION
 FROM rust:${RUST_VERSION}-slim-bullseye
 
 RUN apt-get update \
-  && apt-get install -y g++ cmake pkg-config libclang1-9 llvm-9 libsasl2-dev libssl-dev \
+  && apt-get install -y g++ cmake pkg-config libclang1-9 llvm-9 libsasl2-dev libssl-dev zlib1g-dev zlib1g \
```

Comment:
Nit: `zlib1g-dev` depends on `zlib1g` (most `-dev` packages depend on the non-dev package) so you really only need the `-dev` one.